map style

Hardikkhanesa opened the pull request:
map style change from hybrid and added medical hospitals,zoom in for 3 seconds by another 2 muskeeteers

Hardikkhanesa committed:
map style

map style change from hybrid and added medical hospitals,zoom in for 3 seconds by another 2 muskeeteers

vddesai1998 commented:
Have you run code locally? It seems you are adding a new script.js file in the root directory of the project instead of that you should change the existing script.js file which resides in `static` folder

vddesai1998 commented:
Run code locally and attach screenshot in PR description

Hardikkhanesa commented:
I Have Run Code locally and changes on github file i don't know how to do.

I and avinash added all hospitals in the map with blue colors and set a
time of 3 second to zoom in - out.

On 11 March 2018 at 14:41, Vishal Desai <notifications@github.com> wrote:

> Run code locally and attach screenshot in PR description
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vddesai1998/Disease-Map/pull/1#issuecomment-372100510>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AY3eJpS1rpxLyTjb5nG8sSBSHiEjEEu6ks5tdOpEgaJpZM4Slk73>
> .
>

vddesai1998 commented:
Send the screenshot

vddesai1998 commented:
You have two script.js file in your fork ☺️  
remove script.js from root folder and change the one which is in static folder
then add it to git, commit the changes then push

Hardikkhanesa commented:
![screenshot 31](https://user-images.githubusercontent.com/26074662/37253264-8a6ea48a-2555-11e8-99c0-7806b372e8e3.png)
![screenshot 32](https://user-images.githubusercontent.com/26074662/37253268-8e9ee204-2555-11e8-86af-395952248d1f.png)

Hardikkhanesa commented:
Blue parts are Hospitals and also Zoom in and out option in 3 second time is added

Hardikkhanesa committed:
map style second time

I have not really used right way but i have found another copy-paste way to add this file

vddesai1871 commented on script.js:
```diff
@@ -0,0 +1,268 @@
+/*
```
Remove this file

vddesai1871 commented on static/script.js:
```diff
@@ -24,7 +24,7 @@ $(function() {
     // https://developers.google.com/maps/documentation/javascript/styling
     var styles = [
-        // hide Google's labels
+     /*   // hide Google's labels
```
We are not removing this functionality. We have to make other label's invisible. You can't simply comment out this piece of code.

vddesai1871 replied:
We don't want labels of parks, museums etc.  visible

vddesai1871 commented on static/script.js:
```diff
@@ -39,18 +39,34 @@ $(function() {
-        disableDefaultUI: true,
-        mapTypeId:google.maps.MapTypeId.HYBRID,
+        disableDefaultUI: false,
+        mapTypeId:google.maps.MapTypeId.ROADMAP,
```
We do not need roads and other labels on our map

vddesai1871 commented on static/script.js:
```diff
@@ -140,7 +156,11 @@ function addMarker(data)
         google.maps.event.addListener(marker, 'click', function() {
             showInfo(marker, content);
-		});
+            var pos = map.getZoom(); 
```
LGTM

Hardikkhanesa commented:
its okay but just we dont want that HYBRID in our display

On 11 March 2018 at 18:19, Vishal Desai <notifications@github.com> wrote:

> *@vddesai1998* commented on this pull request.
> ------------------------------
>
> In static/script.js
> <https://github.com/vddesai1998/Disease-Map/pull/1#discussion_r173653302>:
>
> > @@ -24,7 +24,7 @@ $(function() {
>      // styles for map
>      // https://developers.google.com/maps/documentation/javascript/styling
>      var styles = [
> -        // hide Google's labels
> +     /*   // hide Google's labels
>
> We don't want labels of parks, museums etc. visible
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vddesai1998/Disease-Map/pull/1#discussion_r173653302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AY3eJue02pV0mGHem0UX36Z1J2znUBWRks5tdR1egaJpZM4Slk73>
> .
>

vddesai1998 commented:
Then what do you want? Roads? and make suggested changes so I can merge this PR

Hardikkhanesa committed:
Delete script.js
